src/edu/asu/plp/tool/prototype/Main.java

```diff
@@ -413,45 +463,843 @@ private ProjectExplorerTree createProjectTree()
+		
+		button = new ImageView("toolbar_program.png");
+		listener = (event) -> console.println("Program Project Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		buttons.add(new Separator(Orientation.VERTICAL));
+		
+		button = new ImageView("toolbar_step.png");
+		listener = (event) -> console.println("Step Through Project Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		button = new ImageView("toolbar_run.png");
+		listener = (event) -> console.println("Run Project Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		button = new ImageView("toolbar_reset.png");
+		listener = (event) -> console.println("Reset Sim Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		button = new ImageView("toolbar_remote.png");
+		listener = (event) -> console.println("Floating Sim Control Window Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		buttons.add(new Separator(Orientation.VERTICAL));
+		
+		// I Think we are putting all these buttons onto one page
+		// but until that happens I laid them all out, just in case
+		
+		button = new ImageView("toolbar_cpu.png");
+		listener = (event) -> console.println("CPU View");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		button = new ImageView("toolbar_watcher.png");
+		listener = (event) -> console.println("Watcher Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		button = new ImageView("toolbar_sim_leds.png");
+		listener = (event) -> console.println("LED's Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		button = new ImageView("toolbar_sim_switches.png");
+		listener = (event) -> console.println("Switches Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		button = new ImageView("toolbar_sim_7segments.png");
+		listener = (event) -> console.println("7 Seg Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		button = new ImageView("toolbar_sim_uart.png");
+		listener = (event) -> console.println("UART Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		button = new ImageView("toolbar_sim_vga.png");
+		listener = (event) -> console.println("VGA Clicked");
 		button.setOnMouseClicked(listener);
 		buttons.add(button);
 		
+		button = new ImageView("toolbar_sim_plpid.png");
+		listener = (event) -> console.println("PLPID Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		button = new ImageView("toolbar_sim_gpio.png");
+		listener = (event) -> console.println("GPIO Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		button = new ImageView("toolbar_exclamation.png");
+		listener = (event) -> console.println("Interupt Clicked");
+		button.setOnMouseClicked(listener);
+		buttons.add(button);
+		
+		for (int x = 9; x <= 23; x++)
```

Comment:
In fact, just do the set. constants in this case are really dirty and prone to errors in the future.